app/config/mq-config.js

```diff
@@ -62,6 +62,13 @@ const mqSchema = joi.object({
     username: joi.string(),
     password: joi.string(),
     type: joi.string()
+  },
+  responseEligibilityQueue: {
+    name: joi.string().default('ffc-sfi-eligibility-check-response'),
```

Comment:
I wouldn't add the name property, it defaults to the queue name anyway so explicitly setting it again I don't think adds any value